protostar/testing/test_config.py

```diff
@@ -13,6 +13,11 @@
 from .testing_seed import Seed, random_seed
 
 
+class ExampleData:
+    def __init__(self):
+        self.mapping = {}
+
+
```

Comment:
[Design] Why is this class necessary?

Reply:
you can use `PythonData` type alias here instead

Reply:
It's not, will be removed

Reply:
[Remove ExampleData class](https://github.com/software-mansion/protostar/pull/947/commits/7da6be7293f367eef033fe97e6cff9827f102998)

Reply:
use `PythonData` instead of `dict`

Reply:
> you can use `PythonData` type alias here instead, see my other review comment

Thanks for the suggestion but this was really redundant.

Reply:
[allow only keyword based arguments](https://github.com/software-mansion/protostar/pull/947/commits/ca556714d1697625088c577d841cc52c4300b044)